### PR TITLE
[Doc Feature] [Actions in Microsoft 365][3380146]

### PR DESCRIPTION
**Pull request by mobajemu-msft**
*  stand-up Overview and How-to page for Actions

### CODE REVIEW

**Comment by erikadoyle** [comment]
@v-bvishnu could you please help merge this into the feature branch (doc-feature-actions-m365)? From there we'll pass the baton to your team for next steps. /Cc @surbhigupta12  Thanks!

**Comment by mobajemu-msft** [comment]
@v-bvishnu Thank you for your recent changes but I have reverted most of them. It is best if the how-to section you added to the overview doc remains in the how-to doc, the overview should give general guidance what Action is and the how-to should have instruction for how to build it. I was able to pull in all your other changes and thank you for those.

File: msteams-platform/m365-apps/actions.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+ms.topic: Conceptual
+ms.subservice: m365apps
+---
+# Actions in Microsoft 365
+
+> [!NOTE]
+> The Actions in Microsoft 365 is only available for Microsoft 365 app on web and desktop clients.
+> The Actions in Microsoft 365 is available in public developer preview.
+
+Actions seamlessly integrate your app into the user's workflow, ensuring effortless discovery and smooth interaction with their content. By guiding users to your app based on their intent and contextual content, Actions facilitate efficient task completion. Which integration increases your app's visibility and engagement with minimal development effort.
```
Comment (erikadoyle) [inline msteams-platform/m365-apps/actions.md:16]: "seamlessly" and "effortless" might be overselling this a bit...suggest dropping those adjectives. Also "Which integration increases your app's visibility and engagement with minimal development effort." . . . I don't think that's a sentence?

File: msteams-platform/TOC.yml [hunk 2]
```diff
@@ -792,6 +792,14 @@
       - name: Build Adaptive Card-based Loop component
         href: m365-apps/cards-loop-component.md
         displayName: loop component, weburl, metadata
+  - name: M365 Actions
```
Comment (erikadoyle) [inline msteams-platform/TOC.yml:795]: Global comment: Always spell out Microsoft 365 ("M365" isn't sanctioned by branding for public comms)

File: msteams-platform/m365-apps/actions.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+Actions enhance user's productivity by streamlining task completion and reducing the need for context switching across various Microsoft 365 applications.
+
+## Key benefits
+
+1. Users can accomplish tasks more efficiently.
+1. Seamlessly integrates your app into users workflow, increasing app's visibility and user engagement.
+1. Users can take immediate action on content files through your app, expanding the range of interactions they can have with their content.
```
Comment (erikadoyle) [inline msteams-platform/m365-apps/actions.md:24]: This para seems redundant to the intro above; can probably cut

File: msteams-platform/TOC.yml (hunk 2 above)
Comment (erikadoyle) [reply msteams-platform/TOC.yml:795]: Also I think the spacing is misaligned here, so the new articles appear orphaned rather than hooked into the TOC.

**Review by erikadoyle — CHANGES_REQUESTED** [review]
Let's work w/Surbhi and Vishnu on merging the two drafts + including some missing sections from Irene's doc. Meeting forthcoming--

**Comment by surbhigupta12** [comment]
Closing this PR.

**Pull request closed without merging**